ci/deploy_credentials.yaml

```diff
@@ -0,0 +1,34 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-questionnaire-runner-repo
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  EQ_KEYS_FILE:
+  EQ_SECRETS_FILE:
+  PROJECT_ID:
+  REGION:
```

Comment:
Minor, but the `EQ_` ones should be at the bottom. The other environment variables are used directly in the task before the EQ ones that are passed to the script.